src/index.js

```diff
@@ -452,27 +452,40 @@ class LocalstackPlugin {
       return containerID;
     }
 
+    const startContainer = () => {
+      this.log('Starting LocalStack in Docker. This can take a while.');
+      const cwd = process.cwd();
+      const env = this.clone(process.env);
+      env.DEBUG = '1';
+      env.LAMBDA_EXECUTOR = env.LAMBDA_EXECUTOR || 'docker';
+      env.LAMBDA_REMOTE_DOCKER = env.LAMBDA_REMOTE_DOCKER || '0';
+      env.DOCKER_FLAGS = (env.DOCKER_FLAGS || '') + ` -d -v ${cwd}:${cwd}`;
+      env.START_WEB = env.START_WEB || '0';
+      const maxBuffer = (+env.EXEC_MAXBUFFER)||50*1000*1000; // 50mb buffer to handle output
+      if (this.shouldRunDockerSudo()) {
+        env.DOCKER_CMD = 'sudo docker';
+      }
+      const options = {env: env, maxBuffer};
+      return exec('localstack start', options).then(getContainer)
+          .then((containerID) => addNetworks(containerID))
+          .then((containerID) => checkStatus(containerID));
+    }
+
+    const startCompose = () => {
+      return exec(`docker-compose -f ${this.config.docker.compose_file} `).then(getContainer)
```

Comment:
Please add a log message here, so it's easier to debug in future for any support cases or bug reports 😄 

I think this should be ```exec(`docker-compose -f ${this.config.docker.compose_file} up -d`).then(getContainer)```?
the `up` would actually start the docker compose, and `-d` is required so that we sls script will finish.